### DISCUSSION TITLE
Manifest android:minSdkVersion="24"

### DISCUSSION THREAD

**Discussion by farhazulmullick-pw** [KevinnZou/compose-webview-multiplatform#76] [General]
Any specific reason why library marks android.minSdk="24" in properties.gradle ?

**Comment by KevinnZou** [comment]
@farhazulmullick-pw Thanks for your feedback! Honestly, it is just taken from the JetBrains template. Which version do you need?

**Reply by farhazulmullick-pw** [reply]
@KevinnZou we are trying to support at least sdk = 23.
Thanks.
